srcpkgs/OpenRCT2/patches/0001-no-werror.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 2c99d9d41..90b12375a 100644
+--- CMakeLists.txt
++++ CMakeLists.txt
+@@ -260,7 +260,7 @@ else ()
+ 
+     # Compiler flags
+     set(DEBUG_LEVEL 0 CACHE STRING "Select debug level for compilation. Use value in range 0–3.")
+-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fstrict-aliasing -Werror -Wundef -Wmissing-declarations -Winit-self -Wall -Wextra -Wshadow")
```

Comment:
You can try to use `vsed` to remove this instead, to avoida patch that may have to be maintained. Also, we should ask upstream to remove it.